pkg/handlers/user/tenant.go

```diff
@@ -2,28 +2,93 @@ package user
 
 import (
 	"context"
+	"fmt"
+	"os"
 
-	auth0 "github.com/aserto-dev/aserto/pkg/auth0/api"
+	"github.com/aserto-dev/aserto/pkg/cc"
+	"github.com/aserto-dev/aserto/pkg/cc/config"
 	"github.com/aserto-dev/aserto/pkg/client/tenant"
+	"github.com/aserto-dev/aserto/pkg/keyring"
 	"github.com/aserto-dev/go-grpc/aserto/api/v1"
 	"github.com/aserto-dev/go-grpc/aserto/tenant/account/v1"
 	"github.com/aserto-dev/go-grpc/aserto/tenant/connection/v1"
+	"gopkg.in/yaml.v2"
 
 	"github.com/pkg/errors"
 )
 
-func getTenantID(ctx context.Context, client *tenant.Client, token *auth0.Token) error {
+func getTenantID(ctx context.Context, c *cc.CommonCtx, client *tenant.Client, userIdentity string) (string, error) {
 	resp, err := client.Account.GetAccount(ctx, &account.GetAccountRequest{})
 	if err != nil {
-		return errors.Wrapf(err, "get account")
+		return "", errors.Wrapf(err, "get account")
 	}
 
-	token.TenantID = resp.Result.DefaultTenant
+	if err := writeContexts(c, resp.Result.Tenants, resp.Result.DefaultTenant, userIdentity); err != nil {
+		return "", err
+	}
+
+	return resp.Result.DefaultTenant, nil
+}
+
+func writeContexts(c *cc.CommonCtx, tenants []*api.Tenant, defaultTenant, userIdentity string) error {
+	cfgSymlinkPath, err := config.GetSymlinkConfigPath()
+	if err != nil {
+		return err
+	}
+	configPath := fmt.Sprintf("%s-%s.yaml", cfgSymlinkPath, userIdentity)
+
+	cfg := &config.Config{}
+	if config.FileExists(cfgSymlinkPath) {
+		//user already loggedin
+		currentUserFilePath, err := os.Readlink(cfgSymlinkPath)
```

Comment:
You shouldn't have to explicitly read the link to open the file. You can just open the symlink path.